### PR TITLE
fix(ui-search): Handle popup blockers for URL opening

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Refs: [JAVASCRIPT-53A](https://sentry.io/share/issue/4d9dd7db496640eaa25b31abb215844c/)

### CODE REVIEW

File: src/sentry/static/sentry/app/components/search/index.jsx [hunk 1]
```diff
@@ -94,6 +95,13 @@ class Search extends React.Component {
 
     if (to.startsWith('http')) {
       const open = window.open();
+      if (open === null) {
+        IndicatorStore.addError(
+          t('Unable to open search result (a popup blocker may have caused this).')
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/search/index.jsx]: I have some advice on how to do this without being blocked and avoiding `window.open`; we could talk offline.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/search/index.jsx]: Sure. I looked at a few solutions but couldn't find one that would open in a new tab.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/search/index.jsx]: I'm not going to get too involved on this one @benvinegar. We can circle back at some point and turn these into anchors and look closer into the key handling stuff.

File: src/sentry/static/sentry/app/components/search/index.jsx [hunk 2]
```diff
@@ -94,6 +95,13 @@ class Search extends React.Component {
 
     if (to.startsWith('http')) {
       const open = window.open();
+      if (open === null) {
+        IndicatorStore.addError(
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/search/index.jsx]: can we use the `addErrorMessage` action

**Review by billyvg — APPROVED** [review]

**Pull request merged**